api_v3/lib/types/KalturaUiConf.php

```diff
@@ -181,8 +197,11 @@ class KalturaUiConf extends KalturaObject implements IFilterable
 		"updatedAt", 
 		"html5Url",
 		"config",
-        "version",
+		"version",
 		"partnerTags",
+		"v2tov7id",
+		"v2tov7Approved",
+		"v2tov7ShouldTranslatePlugins"
```

Comment:
Need to validate that those values are set only on V2 UCONF, and validate the each param comes after the ID